extensions/hibernate-search-orm-elasticsearch/deployment/src/main/java/io/quarkus/hibernate/search/orm/elasticsearch/deployment/HibernateSearchElasticsearchProcessor.java

```diff
@@ -161,8 +148,19 @@ private void buildForPersistenceUnit(HibernateSearchElasticsearchRecorder record
         }
 
         configuredPersistenceUnits
-                .produce(new HibernateSearchElasticsearchPersistenceUnitConfiguredBuildItem(persistenceUnitName, puConfig,
-                        backendNamesForIndexedEntities, backendAndIndexNamesForSearchExtensions));
+                .produce(new HibernateSearchElasticsearchPersistenceUnitConfiguredBuildItem(
```

Comment:
I've looked at the failing test... maybe we should fail early (`puConfig== null` ?), if there's no config for a PU, since we know that we need at least some configuration to be present.

Reply:
For now I'll just add a null-check, but yes it would be a good idea to move the version check to build time.